### ISSUE TITLE
Wrong data in DB after updating of entry with more caps

### ISSUE THREAD

**Issue by s09bQ5** [SaschaWillems/glCapsViewer#35]
Yesterday I uploaded https://opengl.gpuinfo.org/displayreport.php?id=8670. I quickly noticed that it was missing even the most basic capabilities. After fixing the bug in #34 I used the application to update the database entry. But the entry shown on the website now contains wrong data.
The database dropped 11 capabilities that the application was able to read out, invented 8 capabilities that the application did not read out and messed up the assignment of all values. It looks like the values have the correct order on the website, but are shifted by a few rows. The four "n/a" values from the xml are missing.

The missing capabilities are
- GL_MAX_TEXTURE_SIZE
- GL_MAX_VIEWPORT_DIMS[0]
- GL_MAX_COMBINED_TEXTURE_IMAGE_UNITS
- GL_NUM_COMPRESSED_TEXTURE_FORMATS_ARB
- GL_MAX_TEXTURE_IMAGE_UNITS
- GL_MAX_VARYING_FLOATS
- GL_MAX_VERTEX_ATTRIBS
- GL_MAX_VERTEX_TEXTURE_IMAGE_UNITS
- GL_MAX_COLOR_TEXTURE_SAMPLES
- GL_MAX_DEPTH_TEXTURE_SAMPLES
- GL_MAX_INTEGER_SAMPLES

The "invented" capabilities are
- GL_MAX_COLOR_ATTACHMENTS
- GL_MAX_ARRAY_TEXTURE_LAYERS
- GL_MAX_VERTEX_UNIFORM_VECTORS
- GL_MAX_FRAGMENT_UNIFORM_VECTORS
- GL_MIN_PROGRAM_TEXEL_OFFSET
- GL_MAX_SUBROUTINES
- GL_MAX_GEOMETRY_UNIFORM_BLOCKS
- GL_MAX_VIEWPORTS

I have attached the saved xml: [glCapsViewer_Report.xml.txt](https://github.com/SaschaWillems/glCapsViewer/files/10833306/glCapsViewer_Report.xml.txt)

**Comment by SaschaWillems** [comment]
Yeah, that bug was introduced with #28. I actually saw that the caps.xml was no longer downloaded, but didn't test it  myself.

I'll probably just delete the whole report, so you can reupload.

**Comment by s09bQ5** [comment]
> I'll probably just delete the whole report, so you can reupload.

Go ahead, I still have access to that device. But this will of course not fix the underlying problem.
